src/dft/backends/rocfft/descriptor.cpp

```diff
@@ -22,9 +22,7 @@
 
 #include "oneapi/mkl/dft/detail/rocfft/onemkl_dft_rocfft.hpp"
 
-namespace oneapi {
-namespace mkl {
-namespace dft {
+namespace oneapi::mkl::dft::detail {
```

Comment:
It seems the namespace in this file is not consistent in all backends.
With this change rocFFT, oneMKL CPU, oneMKL GPU have the oneapi::mkl::dft::detail namespace but cuFFT and portFFT have oneapi::mkl::dft namespace.
I know this is unrelated to the original issue, but since you changed one of them, could you please make them consistent across all backends?

Reply:
Thank you for your review @lhuot 
No problem, I updated them in [aed1f63](https://github.com/oneapi-src/oneMKL/pull/563/commits/aed1f63fea977b65a5c2da785b5405e4d1ae8850)